### DISCUSSION TITLE
Reintroduce documentation for support of promise-exporting configuration files?

### DISCUSSION THREAD

**Discussion by mightyiam** [eslint/eslint#19137] [Ideas]
Documentation for this feature has been [removed](https://github.com/eslint/eslint/commit/8792464ee7956af82dab582ca9ee59da596a608e#diff-d6b69160f1f619e34db42e20d9eb94e89630e5a9bd55f41e3d159575eea6c79dL49-L62). Perhaps that was intentional :shrug: but I don't know.

Perhaps it should be reintroduced?

**Comment by nzakas** [comment]
It was removed because that pattern isn't necessary anymore. That was from a time when we only supported `eslint.config.js` but now you can do `eslint.config.mjs` to use `import`.

**Reply by mightyiam** [reply]
You're right. Strictly speaking, it isn't necessary. Anyway, I'm not the user — @rostislav-simonik is.
